### ISSUE TITLE
broken link

### ISSUE THREAD

**Issue by Buliwyfa** [humhub/humhub-modules-devtools#4]
Tutorials > My first Module ( Create your first HumHub module ) 

Gives a

**Comment by byndstd** [comment]
I'm getting the same error

**Comment by funkycram** [comment]
The `controller/TutorialController.php` is missing

**Comment by buddh4** [comment]
There is no tutorial available yet. Perhaps we should add something simple any ideas for a small tutorial modul, perhaps only a sidebar snippet or something?

**Comment by funkycram** [comment]
Yes! Or maybe a very simple module like https://github.com/humhub-contrib/examples

**Comment by buddh4** [comment]
I think such empty examples should rather be added as additional templates to the gii module generator.

**Comment by luke-** [comment]
Here is also a more recent example: https://github.com/humhub/humhub-modules-example-basic
